src/components/common/TechnologyLogos/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import withStyles from 'react-jss';
+import cx from 'classnames';
+import { withPrefix } from 'gatsby-link';
+
+import technologyBrands from '../../../data/technologyBrands.json';
+
```

Comment:
With a few of internal imports, it's better to group them so this empty line is unnecessary.